File: MantenimientoDaxDirectorio/MantenimientoDirectorio/lisDir.cs
using System;
using Microsoft.VisualBasic;

namespace directMnt
{
	public class lisDir
	{
		public void MallaDir(string queTipo = "")
		{
			var PROG = new AdcDir();
			try
			{
				Module1.Mainn();
				PROG.tipo = queTipo;
				PROG.Show();
			}
			// PROG.Dispose()
			catch (Exception ee)
			{
				Interaction.MsgBox("Excepción mallaDir: " + ee.Message);
			}
		}
	}
}

File: MantenimientoDaxDirectorio/MantenimientoDirectorio/lisDir.vb
Option Strict Off
Option Explicit On
Public Class lisDir
    Public Sub MallaDir(Optional queTipo As String = "")
        Dim PROG As New AdcDir
        Try
            Mainn()
            PROG.tipo = queTipo
            PROG.Show()
            'PROG.Dispose()
        Catch ee As Exception
            MsgBox("Excepción mallaDir: " & ee.Message)
        End Try
    End Sub
End Class
